docs/content/docs/start/tutorial.md

```diff
@@ -99,24 +99,30 @@ start().catch((error) => console.error(error));
 
 ## Write the dice view
 
-The Fluid Framework is view framework agnostic and works well with React, Vue, Angular and web components. This example uses standard HTML/DOM methods to render a view. You can see this example, as well as examples of the previously mentioned frameworks, in our [HelloWorld repo](https://github.com/microsoft/FluidHelloWorld).
+The Fluid Framework is view framework agnostic and works well with React, Vue, Angular and web components. This example uses standard HTML/DOM methods to render a view. You can see examples of the previously mentioned frameworks in the [Fluid Examples repo](https://github.com/microsoft/FluidExamples/tree/main/multi-framework-diceroller).
 
 ### Start with a static view
 
 It is simplest to create the view using local data without Fluid, then add Fluid by changing some key pieces of the app. This tutorial uses this approach.
 
-The `renderDiceRoller` function below takes an HTML element to attach to, and displays a working dice roller with a random dice value each time the "Roll" button is clicked. Note that the included code snippets omit the styles for brevity.
+The `renderDiceRoller` function appends the `diceTemplate` to the passed in HTML element, and creates a working dice roller with a random dice value each time the "Roll" button is clicked. The `diceMap` will be used in the next few steps.
 
 ```js
+const diceTemplate = document.createElement("template");
+
+diceTemplate.innerHTML = `
+  <div class="wrapper">
+    <div class="dice"></div>
+    <button class="roll"> Roll </button>
+  </div>
+`
```

Comment:
Man, we _really_ need to get twoslash working! :)

Reply:
what does two slash do here?